docs/articles/documentation/using-testcafe/programming-interface/Runner.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: docs
+title: Runner Class
+permalink: /documentation/using-testcafe/programming-interface/Runner/
+---
+# Runner Class
+
+An object that configures and launches test runs.
+
+Created by the [testCafe.createRunner](/testcafe/documentation/using-testcafe/programming-interface/TestCafe/#createRunner) function.
+
+#### Example
+
+```js
+var createTestCafe = require('testcafe');
+var testCafe = await createTestCafe('localhost', 1337, 1338);
+
+testCafe
+    .createRunner()
+    .src(['/tests/fixture1.js', '/tests/func/fixture3.js'])
+    .browsers(['chrome', 'firefox'])
+    .run()
+    .then(failed => console.log('Tests failed: ' + failed));
+```
+
+## Methods
+
+### <a class="anchor" name="src"></a>src(source) → this
+
+### src([source, ...]) → this
+
+Configures the test runner to run tests from the specified files.
+
+Parameter | Type   | Description
+--------- | ------ | -----------------------------------------------------
+`source`  | String | The relative or absolute path to a test fixture file.
```

Comment:
Look at the following lines
```
### <a class="anchor" name="src"></a>src(source) → this

### src([source, ...]) → this
```
They already imply that you can pass a `source` or an array of `source`
Or do you think that `src([source, ...])` is non-standard and confused with optional arguments?